Library/Homebrew/os/linux/elf.rb

```diff
@@ -172,6 +188,19 @@ def needed_libraries_using_readelf(path)
     end
   end
 
+  def patcher
+    @patcher ||= begin
+      begin
+        Homebrew.setup_gem_environment!
+        require "patchelf"
+      rescue LoadError
+        Homebrew.install_bundler_gems!
```

Comment:
we could have clean code here(just `require "patchelf"`)  by placing the install_bundler_gems in globals.rb,
tradeoff  would be an  extra `require 'utils/gems'` in globals.rb.

Reply:
`Homebrew.install_bundler_gems!` is a no-op if they aren't installed. Call it unconditionally before  `require "patchelf"`